sympy/parsing/sympy_parser.py

```diff
@@ -385,25 +389,28 @@ def _split_symbols(tokens, local_dict, global_dict):
                 split_previous=False
                 continue
             split_previous=False
-            if tok[0] == NAME and tok[1] == 'Symbol':
+            if tok[0] == NAME and tok[1] in ['Symbol', 'Function']:
                 split = True
             elif split and tok[0] == NAME:
                 symbol = tok[1][1:-1]
                 if predicate(symbol):
-                    for char in symbol:
+                    tok_type = result[-2][1]  # Symbol or Function
+                    del result[-2:]  # Get rid of the call to Symbol
+                    for char in symbol[:-1]:
                         if char in local_dict or char in global_dict:
-                            # Get rid of the call to Symbol
-                            del result[-2:]
-                            result.extend([(NAME, "%s" % char),
-                                           (NAME, 'Symbol'), (OP, '(')])
+                            result.extend([(NAME, "%s" % char)])
```

Comment:
This is the only line that was added that is not hit by tests of `parsing` according to my analysis. The following hits it:
```
>>> t=transformations = standard_transformations + (split_symbols, implicit_multiplication,)
>>> parse_expr('yx', local_dict={'x':w},transformations=t)
w*y
```